
<launch>
	<node pkg="gatlin" type="gatlin_mott.py" name="gatlin_mott" output="screen" ></node>
</launch>